packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -755,4 +758,35 @@ router.post(
   })
 );
 
+/**
+ * POST /internal/patient/:id/consolidated
+ *
+ * Kicks off patient discovery for the given patient on both CQ and CW.
+ * @param req.query.cxId The customer ID.
+ * @param req.params.id The patient ID.
+ * @param req.query.rerunPdOnNewDemographics Optional. Indicates whether to use demo augmentation on this PD run.
+ */
+router.post(
+  "/:id/consolidated",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const id = getFromParamsOrFail("id", req);
+    const patient = await getPatientOrFail({ cxId, id });
+
+    //TODO Parse body?
```

Comment:
Were leaving this comment?

Reply:
added a schema validation and remove this